### PR TITLE
add play button

### PR DESCRIPTION
**Pull request by hamirmahal**

### CODE REVIEW

File: src/Sound.js [hunk 1]
```diff
@@ -35,13 +35,18 @@ export default function Sound(props) {
...
+  const [isPlaying, setIsPlaying] = useState(false);
   const [similarSounds, setSimilarSounds] = useState([]);
   // 0 => loading
   // 1 => loading succeeded
   // 2 => loading failed
   const [loadingState, setLoadingState] = useState(0);
   const { id } = useParams();
 
+  const handlePlayingAndPausing = () => {
+    setIsPlaying(!isPlaying);
```
Comment (amilajack) [inline src/Sound.js:47]: `setIsPlaying` needs to be called when progress has finished. You can hook into this with the `onFinish` prop:

https://github.com/amilajack/react-wavesurfer.js/blob/master/src/index.tsx#L114

**Review by amilajack — CHANGES_REQUESTED** [review]

File: src/Sound.js (hunk 1 above)
Comment (hamirmahal) [reply src/Sound.js:47]: Working on this right now. Is that to automatically change the button when a sound finishes?

Comment (amilajack) [reply src/Sound.js]: Yup, that's correct

**Comment by amilajack** [comment]
@hamirmahal I think you might have accidentally closed the PR

**Comment by hamirmahal** [comment]
> @hamirmahal I think you might have accidentally closed the PR

I did. I was able to figure out the play/pause button issue and made a commit, but I'm not sure how to make that commit appear here, or if it's even possible. I'm in the process of creating a new PR, but I haven't hit the "Create pull request" button, yet.

Screen Shot 2020-09-25 at 8 32 34 PM

File: src/Sound.js (hunk 1 above)
Comment (hamirmahal) [reply src/Sound.js]: Sounds good. It appears to work, now. You can see what I think is a fully working commit in [the `add-play-button` branch](https://github.com/amilajack/freesound-player/tree/add-play-button).

**Comment by hamirmahal** [comment]
This pull request closes #54 if we choose to merge it.

**Review by amilajack — APPROVED** [review]
Nice!

**Pull request merged**